### DISCUSSION TITLE
Parsing error: Unexpected token {

### DISCUSSION THREAD

**Discussion by LOUSANPANG** [nuxt/eslint-config#444] [Q&A]
## package.json

![image](https://github.com/nuxt/eslint/assets/36870764/bca8c5db-ecca-4861-b012-d4b08c76bf12)

## eslint.config.mjs

![image](https://github.com/nuxt/eslint/assets/36870764/ae29dac0-d55d-4744-993b-a7867e5ff1cb)

## nuxt.config.js

![image](https://github.com/nuxt/eslint/assets/36870764/ef45bfe2-a4d0-4f7f-9aa9-1b0172f47029)

## Error info

![image](https://github.com/nuxt/eslint/assets/36870764/1c3d7647-6f7c-4c9e-a397-10e0c72e8d5c)

![image](https://github.com/nuxt/eslint/assets/36870764/5b75961f-c744-4286-b54e-440bcd75dafe)

**Comment by egor1k** [comment]
I have the same problem, on version @nuxt/eslint 0.4.0

**Comment by AloisSeckar** [comment]
I also come across this when updating to `@nuxt/eslint 0.4.0` today. Version `@nuxt/eslint 0.3.13` works fine.

Latest version has problems with `type` keyword.

Examples of my parsing errors:

![image](https://github.com/user-attachments/assets/2c7acb34-a907-4276-91a7-0e0ef524de36)

![image](https://github.com/user-attachments/assets/63e1a392-fc4f-41ee-9bdc-7f498acb95dc)

**Comment by AloisSeckar** [comment]
In case someone comes across this later - current solution is to add explicit `typescript` dependency to your `package.json` as mentioned in the docs since 9f8655e921a64382fb82f8c8a642ed8bee2822ad

**Reply by mhpsy** [reply]
thank bro

**Reply by alebar42** [reply]
Legend 😅

**Reply by AuroraTea** [reply]
You are 👍🏻👍🏻👍🏻

**Reply by www-wanglong** [reply]
👍🏻👍🏻👍🏻
